database/qb/utility.go

```diff
@@ -0,0 +1,17 @@
+package qb
+
+import (
+	"fmt"
+	"strings"
+)
+
+// InsertSQLParameters is a utility function that inserts parameters into parameterized SQL.
+func InsertSQLParameters(sql string, params []interface{}) string {
```

Comment:
this will allow for SQL Injection attacks. Parameterized queries make injection impossible. If we cannot parameterize we will need to have this function sanitize the inputs. And we should have some tests on this function

Reply:
We don't actually need this, I will remove it. The SelectExpression interface doesn't need to include a method to get the raw SQL anymore